phpBB/phpbb/user.php

```diff
@@ -236,6 +236,12 @@ function setup($lang_set = false, $style_id = false)
 			FROM ' . STYLES_TABLE . " s
 			WHERE s.style_id = $style_id";
 		$result = $db->sql_query($sql, 3600);
+		if (!$result) {
+			$sql = 'SELECT *
+				FROM ' . STYLES_TABLE . " s
+				LIMIT 1";
+			$result = $db->sql_query($sql, 3600);
```

Comment:
This change makes no sense. If anything, just cast $style_id to int in the original query..